libr/magic/d/default/filesystems

```diff
@@ -118,6 +128,7 @@
 # http://www.visopsys.org/
 # http://partitionlogic.org.uk/
 >309	string	No\ bootable\ partition\ found\r
+>>>>>>> 9a4fe919c644d18bc037335428debbf9699b6074
```

Comment:
more conflicts commited

Reply:
@trufae I will redo everything again, can you put an assign label so that no one else take up this issue?